docs/rust/testing.md

```
@@ -0,0 +1,170 @@
+command that has since been renamed should say so, not quietly assert about a different page.
+
+`help_tree` is the drift test: every command in the tree, depth-first, in one string.
+
+```rust
+#[test]
+fn help_has_not_drifted() {
+    insta::assert_snapshot!(harness::help_tree(Ex::spec(), Page::Long));
+}
+```
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)